rasa/nlu/data_router.py

```diff
@@ -136,152 +163,56 @@ def _create_query_logger(response_log):
             )
             return None
 
-    def _collect_projects(self, project_dir: Text) -> List[Text]:
-        if project_dir and os.path.isdir(project_dir):
-            logger.debug("Listing projects in '{}'.".format(project_dir))
-            projects = os.listdir(project_dir)
-        else:
-            projects = []
-
-        projects.extend(self._list_projects_in_cloud())
-        return projects
-
-    async def _create_project_store(self, project_dir: Text) -> Dict[Text, Any]:
-        default_project = RasaNLUModelConfig.DEFAULT_PROJECT_NAME
+    async def load_model(self, model_dir: Text):
+        if model_dir is None:
+            logger.info("Could not load any model. Use fallback model.")
+            self.nlu_model = NLUModel.fallback_model(self.component_builder)
+            return
 
-        projects = self._collect_projects(project_dir)
-
-        project_store = {}
+        try:
+            if os.path.exists(model_dir):
```

Comment:
`model_dir` or `model_name`?  Surely we are keeping the directory constant now

Also for some bizarre reason, I can not get this statement to resolve to `True` even when passing a valid path (which resolves to true in a regular instance of Python), can you?

Reply:
For me it is working fine.

The idea is that `model_dir` is a path pointing either to a directory containing any number of `tar.gz` model files of to one specific model file. If it is pointing to a directory, the latest model in that directory is taken. I'll rename the variable to `model_path` and add a comment.